src/schema/type/Rating.graphql

```diff
@@ -61,6 +61,17 @@ type Rating implements ThingInterface {
     "https://schema.org/url"
     url: String
 
+    ############################################
+    ### ProvenanceEntityInterface properties ###
+    "http://www.w3.org/ns/prov#wasGeneratedBy"
```

Comment:
got to add these to the jsonld definition file

Reply:
@alastair are you going to add that? You can use this definition (from DigitalDocument.json)

```json
{
  "properties": {
    "wasGeneratedBy": [
      "http://www.w3.org/ns/prov#wasGeneratedBy"
    ],
    "wasDerivedFrom": [
      "http://www.w3.org/ns/prov#wasDerivedFrom"
    ],
    "wasAttributedTo": [
      "http://www.w3.org/ns/prov#wasAttributedTo"
    ],
    "used": [
      "http://www.w3.org/ns/prov#used"
    ]
  }
}
```

Reply:
I just added it and force pushed